### DISCUSSION TITLE
Why is there no sound in the output segment when I input a movie？

### DISCUSSION THREAD

**Discussion by MrLiHH** [Breakthrough/PySceneDetect#367] [Q&A] [answered]
When I input a movie (about 2.5 hours), the output clip has no sound. But when I input a short video, there is sound. Looking forward to your reply.

**Comment by Breakthrough** [comment, marked as answer]
You might have to customize the options passed to `ffmpeg` when splitting the input video.  [The default args](https://www.scenedetect.com/docs/0.6.2/cli.html#cmdoption-scenedetect-split-video-a) are currently:

    -map 0 -c:v libx264 -preset veryfast -crf 22 -c:a aac

You may want to manually run `ffmpeg` on a small sample of the input file you are trying to split, and figure out what options to set in `ffmpeg`.  You can then customize `split-video --args` accordingly 😄   This is especially important depending on how many audio tracks there are, or if subtitle tracks are present.

The default options try to preserve compatibility with the majority of input sources to generate a valid video output, but are not that focused on reliable audio output yet.  You may want to set a different audio codec/bitrate, and also set what audio track(s) to include in the output.
